web/src/services/udx/routes.ts

```diff
@@ -103,5 +103,20 @@ export default [
         }
       }
     ]
+  },
+  {
+    path: "/api/:version/s3",
+    method: "get",
+    handler: [
+      checkRequiredParams,
```

Comment:
pls also add checkApiVersion middleware

Reply:
Added 'checkApiVersion' was missed in last commit